earth_enterprise/src/fusion/autoingest/plugins.src/PacketGen.src

```diff
@@ -362,6 +338,8 @@ void PacketGenAssetVersionImplD::UpdateChildren(
                   prevLevelVer.Ref() : SharedString() /* cached blend alpha */));
       }
 
+      std::remove(levInputs.begin(), levInputs.end(), "");
+      std::remove(levInputVers.begin(), levInputVers.end(), nullptr);
```

Comment:
As discussed, the `std::remove` calls here are leaving the vectors with duplicate items at the end. Add `erase` calls to remove the extraneous items.